Frontend/src/ContextMenu.svelte

```diff
@@ -0,0 +1,172 @@
+<script>
+    //NOTE: Add in Font Awesome later
+    let position = { x: 0, y: 0 };
+
+    let menu = { w: 0, h: 0 };
+
+    let browser = { w: 0, h: 0 };
+
+    let showMenu = false;
+    
+    let content;
+
+    function rightClickContextMenu(e){
+        showMenu = true;
+        browser = {
+            w: window.innerWidth,
+            h: window.innerHeight
+        };
+
+        position = {
+            x: e.clientX,
+            y: e.clientY
+        };
+
+
+        // console.log("x: ", position.x );
+        // console.log("y: ", position.y, );
+
+        // if (browser.w -  position.x < menu.w) {}
+        //     position.x = position.x - menu.w
+        // }
+        // if (browser.h -  position.y < menu.h) {
+        //     position.y = position.y - menu.h
+
+
+
+    }
+
+        console.log("w: ", menu.w );
+        console.log("h: ", menu.h, );
+
+    function offMenuClick(e){
+        showMenu  = false;
+    }
```

Comment:
Nothing wrong with this at all, I just want to make sure you know it's okay to shorten stuff like this to reduce bloat if you like.

If offMenuClick is only used one time, you can write it like: `on:click={() => showMenu  = false}`

and if it's used multiple times you could write this in lambda notation: `let offMenuClick = () => showMenu  = false;`